slideshow-doc/scribblings/slideshow/slides.scrbl

```diff
@@ -571,11 +571,11 @@ title. The default is @racket["black"].}
 
 Parameter whose value is a function for assembling slide content into
 a single pict; the assembling function takes a string for the title
-(or @racket[#f]), a separation for the title (if any) and pict, and a
```

Comment:
I don't think "and pict" is a typo.

(It's a separation between two things)

Reply:
But the assembling function gets three things:
* string for the title or `#f`
* separation for the title
* content pict

From reading the text it seems it gets four items: string for the title, separation for the title, pict (no reference to what it could be) and a pict for the slide content. I still think this is a mistake.

My native language is not English, but what's written really doesn't sound right *to me*.